(falco-talon): add 0.2.0 release, update 0.1.0 release installation instructions (#1416)
falcosecurity/falco-website: content/en/blog/falco-talon-v0-2-0/index.md

---

> title: Introducing Falco Talon v0.1.0

Reviewer: ```suggestion
title: Falco Talon v0.2.0
```

date: 2024-11-27
author: Igor Eulalio
slug: falco-talon-v0-2-0
images:

>   - /blog/falco-talon-v0-1-0/images/falco-talon-featured.png

Reviewer: ```suggestion
  - /blog/falco-talon-v0-2-0/images/falco-talon-featured.png
```

tags: ["Talon", "Release"]
---

Today we announce the release of **Falco Talon 0.2.0** 🦅!

Falco Talon 0.2.0 is a minor release that includes new actionners and outputs, add parameters to existing actionners, along one small fix on the check and print commands.

## Features

* Add `gcp:function` actionner
* Add `gcp:gcs` output
* Add `ignore_standalone_pods` parameter for `kubernetes:terminate` actionner
* Allow to wait until the completion of `kubernetes:drain` by configuring `max_wait_period`
* Use smaller image for the `kubernetes:tcpdump` actionner

## Fixes

> * Allow to `check` and `print` commands to work without specifying a `config.yaml`

Reviewer: Can you add more details, like snippets to use the actionners, a screenshot of GC Storage, ...

---
title: Falco Talon v0.2.0
date: 2024-11-27
author: Igor Eulalio
slug: falco-talon-v0-2-0
images:
  - /blog/falco-talon-v0-2-0/images/falco-talon-featured.png
tags: ["Talon", "Release"]
---

Today we announce the release of **Falco Talon 0.2.0** 🦅!

Falco Talon 0.2.0 is a minor release that includes new actionners and outputs, add parameters to existing actionners, along one small fix on the check and print commands.

## Features

> * Add `gcp:function` actionner:
>   - Now users can call GCP function to automate GCP tasks, with authentication and authorization out of the box.
> ![gcp-function](/blog/falco-talon-v0-2-0/images/falco-talon-v0-2-0-1.png)
> * Add `gcp:gcs` output
>   - Now users can send output directly to GCP Google Cloud Storage, same way as s3 and minio existing outputs.
> ![gcp-gcs](/blog/falco-talon-v0-2-0/images/falco-talon-v0-2-0-2.png)
> * Add `ignore_standalone_pods` parameter for `kubernetes:terminate` actionner
> * Allow to wait until the completion of `kubernetes:drain` by configuring `max_wait_period` and `wait_period_excluded_namespaces`
> * Use smaller image for the `kubernetes:tcpdump` actionner

Reviewer: ```suggestion
### New actionner

You can now call GCP functions to automate your tasks within GCP, with authentication and authorization out of the box.

Example:
```yaml
....
```

### New output

The new output `gcs:storage` allows you to upload the resulting artifacts to GCP Google Cloud Storage, like the logs or tcpdump captures.

![gcp-gcs](/blog/falco-talon-v0-2-0/images/falco-talon-v0-2-0-2.png)

### Enhancements

* The actionner `kubernetes:terminate` can terminate the standalone if the parameter `ignore_standalone_pods` is set to `true`
* Talon can wait until the completion of the drain by the actionner `kubernetes:drain` by configuring `max_wait_period` and `wait_period_excluded_namespaces`
* The actionner `kubernetes:tcpdump` uses a smaller size image for a quicker spawn

## Fixes

> * Allow to `check` and `print` commands to work without specifying a `config.yaml`

Reviewer: ```suggestion
An existing `config.yaml` file is not required anymore to check the syntax of your rules files.
```
